site/source/docs/getting_started/downloads.rst

```diff
@@ -150,7 +150,9 @@ Linux
 		
 		# Install Java
 		sudo apt-get install default-jre
-		
+
+  Please note that you will have to change the ``NODE_JS`` attribute of the ``~/.emscripten`` file from ``node`` to ``nodejs`` with certain package managers.
```

Comment:
please use sphinx note syntax (see an example higher above in the same file), ` .. note:: `